content/en/integrations/guide/storage-monitoring-setup.md

```diff
@@ -18,9 +18,37 @@ Access your Storage Monitoring data by navigating to **Infrastructure > Storage
 ### Installation
 
 {{< tabs >}}
+{{% tab "Recommended: Storage Monitoring UI" %}}
+
+The fastest way to set up Storage Monitoring is going to Infrastructure -> Storage Monitoring -> [Add Buckets][https://app.datadoghq.com/storage-monitoring?mConfigure=true]. In the Add Buckets page you can configure multiple S3 buckets for Storage Monitoring in one go.
+
+1. Go to Datadog > Infrastructure > Storage Monitoring.
+
+2. Click [Add Buckets][https://app.datadoghq.com/storage-monitoring?mConfigure=true]
+
+{{< img src="integrations/guide/storage-monitoring/add-buckets.png" alt="Select buckets for enabling Storage Monitoring" responsive="true">}}
+
+3. Enable Amazon S3 Integration and Resource collection for all the AWS accounts you want to monitor.
+
+  Note: For each AWS account that has the S3 buckets you want to monitor, make sure your Datadog IAM role(s) includes the following permissions: `s3:GetObject`, `s3:ListObjects`, `s3:PutInventoryConfiguration`
+
+4. Select the S3 buckets you want to monitor with Storage Monitoring. Tip: You can select buckets from multiple AWS accounts at once.
+
+{{< img src="integrations/guide/storage-monitoring/step-2.png" alt="Select buckets for enabling Storage Monitoring" responsive="true">}}
+
+5. Assign a destination bucket (may already exist, or create a new one in AWS) per region to store S3 inventory reports from the source buckets.
```

Comment:
```suggestion
5. Assign a destination bucket per region to store S3 inventory reports from the source buckets. This can be an existing AWS bucket or a new one.
```